packages/cdktf/lib/tfExpression.ts

```diff
@@ -163,7 +166,7 @@ class Reference extends TFExpression {
   }
 }
 
-export function ref(identifier: string, stack: TerraformStack): IResolvable {
+export function ref(identifier: string, stack?: TerraformStack): IResolvable {
```

Comment:
nice, this simplifies my iterator code 👌

Reply:
Yeah there are a few edge cases where not passing a source stack is appropriate since no cross stack references are supported